modules/rostests/winetests/ntdll/threadpool.c

```diff
@@ -20,71 +20,95 @@
 
 #include "ntdll_test.h"
 
-static HMODULE hntdll = 0;
```

Comment:
Is the stuff in this file a wine sync? Custom mofifications? IMO it should have it's own commit(s).

Reply:
This was a winesync. I'll restructure the commits a bit so it aligns with your suggestion.

Reply:
This is now complete, there's now a commit dedicated to a direct copy and paste of the winetest threadpool.c file from wine-9.7 as asked!